### PR TITLE
Move chrome profile to .dart_tools

### PR DESCRIPTION
**Pull request by grouma**
Closes https://github.com/dart-lang/webdev/issues/297

### CODE REVIEW

File: webdev/lib/src/serve/chrome.dart [hunk 1]
```diff
@@ -62,7 +63,9 @@ class Chrome {
   ///
   /// Each url in [urls] will be loaded in a separate tab.
   static Future<Chrome> start(List<String> urls, {int port}) async {
-    var dataDir = Directory.systemTemp.createTempSync();
+    var dataDir = Directory.fromUri(Uri.parse(p.joinAll(
```
Comment (jakemac53) [inline webdev/lib/src/serve/chrome.dart]: Is there a reason you can't just do `Directory(p.join('.dart_tool', 'webdev', 'chrome_profile'))`?

Comment (grouma) [reply webdev/lib/src/serve/chrome.dart]: Discussed offline. This suggestion is safe under Windows so I'll change.

File: webdev/lib/src/serve/chrome.dart [hunk 2]
```diff
@@ -62,7 +63,9 @@ class Chrome {
   ///
   /// Each url in [urls] will be loaded in a separate tab.
   static Future<Chrome> start(List<String> urls, {int port}) async {
-    var dataDir = Directory.systemTemp.createTempSync();
+    var dataDir = Directory(p.joinAll(
+        [Directory.current.path, '.dart_tool', 'webdev', 'chrome_profile']))
```
Comment (natebosch) [inline webdev/lib/src/serve/chrome.dart]: Have we already made some guarantee that `cwd` is the package root?

Comment (grouma) [reply webdev/lib/src/serve/chrome.dart]: Yes: https://github.com/dart-lang/webdev/blob/master/webdev/lib/src/command/serve_command.dart#L102

Comment (kevmoo) [reply webdev/lib/src/serve/chrome.dart]: NICE!

**Pull request merged**